addons/mail/static/src/new/discuss/discuss.js

```diff
@@ -19,6 +20,9 @@ export class Discuss extends Component {
         this.popover = usePopover();
         this.closePopover = null;
         this.settingsRef = useRef("settings");
+        this.state = useState({
+            activeMode: "",
+        });
```

Comment:
Document the different values, even if just inline comment.
Also the value are best in `kebab-case`, so that it's more readable and easier to read the value at a glance. `pascalCase` string values are hard to read and more error-prone than `kebab-case` equivalent.